install_files/ansible-base/roles/tails-config/files/securedrop_init.py

```diff
@@ -48,7 +50,27 @@
                  '/org/gnome/nautilus/preferences/automatic-decompression',
                  'false'])
 
-# notify the user
+# Set journalist.desktop and source.desktop links as trusted with Nautilus (see
+# https://github.com/freedomofpress/securedrop/issues/2586)
+# set euid and env variables to amnesia user
+os.setresgid(1000, 1000, -1)
+os.setresuid(1000, 1000, -1)
+env = os.environ.copy()
+env['XDG_RUNTIME_DIR'] = '/run/user/1000'
```

Comment:
And then we can use `.format` to demonstrate where the `1000` is coming from: 

`'/run/user/{}'.format(blah)`